pkg/BUILD

```diff
@@ -1,7 +1,3 @@
-package(default_visibility = ["//visibility:public"])
-
-licenses(["notice"])
-
```

Comment:
Not sure how this header got wiped out but you need the public visibility decleration